addons/website_event_track/models/event_event.py

```diff
@@ -50,6 +55,16 @@ def _compute_sponsor_count(self):
         for event in self:
             event.sponsor_count = result.get(event.id, 0)
 
+    @api.depends('event_type_id', 'website_menu')
+    def _compute_website_menu_data(self):
```

Comment:
`_compute_website_data` or `_compute_website_menu_data` ?

Pick one to rule them all!

Reply:
Actually it was intended to make two methods because of some trigger mess . Maybe I can add a small docstring.

Reply:
I said du caca. I updated naming.